core/tee/entry_std.c

```diff
@@ -561,7 +562,22 @@ uint32_t __tee_entry_std(struct optee_msg_arg *arg, uint32_t num_params)
 		break;
 #endif
 #endif
+
+	case OPTEE_MSG_CMD_DO_BOTTOM_HALF:
+		if (notif_async_is_enabled())
+			notif_deliver_event(NOTIF_EVENT_DO_BOTTOM_HALF);
+		else
+			goto err;
```

Comment:
why not `rv = OPTEE_SMC_RETURN_EBADCMD;` straight?

Reply:
I thought it nice to get the "Unknown cmd..." print in such a case.